lib/model/request/auth_token_request.dart

```diff
@@ -11,9 +11,9 @@ const _grantTypePassword = "password";
 class AuthTokenRequest {
   final String email;
   final String password;
-  final String grantType = _grantTypePassword;
-  final String clientId = Env.authClientId;
-  final String clientSecret = Env.authClientSecret;
+  late String grantType = _grantTypePassword;
+  late String clientId = Env.authClientId;
+  late String clientSecret = Env.authClientSecret;
```

Comment:
@AVI5HEK Why not we call them directly in `AuthenticationRepositoryImpl` so we can get rid of defining `late` 🤔